src/components/mainLayout/youtubeEmbed/youtubeEmbed.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import { IYoutubeEmbed } from "../../../types/components/ComponentsTypes";
+
+import fallback from "../../../assets/components/youtubeEmbed/fallback.png";
+
+import style from "./youtubeEmbed.module.scss";
+
+const YoutubeEmbed = ({ embedId }: IYoutubeEmbed) => {
```

Comment:
can you rename folder, files, component and interface titles with "VideoElement"?